### PR TITLE
Update it_it.json

### PR DESCRIPTION
**Pull request by CapoFantasma97**
Fixed a lot of issues and updated the file.

Changes to be in line with the game's official style guide for consistency:
- removed abuse of capitalizations (in Italian we only capitalize the start of a phrase and proper nouns, not common nouns)
- fixed inconsistent usage of prepositions
- fixed some terms being translated differently than vanilla

Other changes:
- fixed A LOT of typos (hoping I caught them all)
- fixed wrong translations
- fixed untranslated strings
- some missing strings were added

### CODE REVIEW

**Comment by CapoFantasma97** [comment]
If you could use it to update 1.19.2 as well and not just 1.19.3 that would be great.

File: common/src/main/resources/assets/adorn/lang/it_it.json [hunk 1]
```diff
@@ -1,1278 +1,1373 @@
...
-  "block.adorn.black_sofa": "Divano Nero",
-  "block.adorn.brown_sofa": "Divano Marrone",
-  "block.adorn.red_sofa": "Divano Rosso",
-  "block.adorn.orange_sofa": "Divano Arancione",
-  "block.adorn.yellow_sofa": "Divano Giallo",
-  "block.adorn.lime_sofa": "Divano Lime",
-  "block.adorn.green_sofa": "Divano Verde",
-  "block.adorn.cyan_sofa": "Divano Ciano",
-  "block.adorn.light_blue_sofa": "Divano Celeste",
+{acacia_chair
```
Comment (Juuxel) [inline common/src/main/resources/assets/adorn/lang/it_it.json]: This probably doesn't belong here:
```suggestion
{
```

Comment (CapoFantasma97) [reply common/src/main/resources/assets/adorn/lang/it_it.json]: Yes sorry lol.

**Comment by CapoFantasma97** [comment]
Good to go?

**Review by Juuxel — APPROVED** [review]
Looks great, thanks!

**Comment by CapoFantasma97** [comment]
> Looks great, thanks!

Epic 👍 Can you also backport it for previous Minecraft versions (e.g. 1.19.2)? Or do I create other push requests?

**Comment by Juuxel** [comment]
I'll backport it to .2 along with all other content changes for the next update.

**Pull request merged**